rac-experiments-v3-hashed-keys.json

```diff
@@ -341,6 +362,81 @@
           ]
         }
       }
+    },
+    "b8536ad873f30e17508b654190368e3a": {
+      "subjectShards": 10000,
+      "overrides": {},
+      "typedOverrides": {},
+      "enabled": true,
+      "rules": [
+        {
+          "allocationKey": "allocation-experiment-1",
+          "conditions": []
+        }
+      ],
+      "allocations": {
+        "allocation-experiment-1": {
+          "percentExposure": 0.4533,
+          "statusQuoVariationKey": "variation-7",
+          "shippedVariationKey": "variation-8",
+          "holdouts": [
+            {
+              "holdoutKey": "holdout-2",
+              "statusQuoShardRange": {
+                "start": 0,
+                "end": 100
+              },
+              "shippedShardRange": {
+                "start": 100,
+                "end": 200
+              }
+            },
+            {
+              "holdoutKey": "holdout-3",
+              "statusQuoShardRange": {
+                "start": 200,
+                "end": 300
+              },
+              "shippedShardRange": {
+                "start": 300,
+                "end": 400
+              }
```

Comment:
Consider making these numbers more random (complete with gaps between them) to reflect our planned strategy

Reply:
Ah yes, good catch!